tests/e2e/apps/hop/app_test.go

```diff
@@ -32,246 +31,245 @@ import (
 	"istio.io/istio/tests/e2e/apps/hop/config"
 )
 
+var seed int64 = 1823543
+
 type testServers struct {
-	gs      *[]*grpc.Server
-	hs      *[]*httptest.Server
-	remotes *[]string
+	gs      []*grpc.Server
+	hs      []*httptest.Server
+	remotes []string
 }
 
 type errorServer struct{}
 
 type handler interface {
-	ServeHTTP(w http.ResponseWriter, r *http.Request)
+	ServeHTTP(w http.ResponseWriter, _ *http.Request)
```

Comment:
(nit): The unused `_` is more appropriate in the implementation on line 50 to show that the variable isn't used.